travelpal/lib/travelpal_web/controllers/session_controller.ex

```diff
@@ -0,0 +1,28 @@
+defmodule TravelpalWeb.SessionController do
+  use TravelpalWeb, :controller
+
+  alias Travelpal.Accounts
+  alias Travelpal.Accounts.User
+
+  def create(conn, %{"username" => username}) do
+    user = Accounts.get_user_by_username(username)
+
+    if user do
+      conn
+      |> put_session(:user_id, user.id)
+      |> put_flash(:info, "Welcome back #{user.username} 🙌")
```

Comment:
Nitpick: can you add a comma after "back" and a period or exclamation mark after the username?
`"Welcome back, #{user.username}! 🙌"`

Reply:
Fixed